Source/init.cpp

```diff
@@ -13,7 +13,7 @@ char diablo_exe_path[MAX_PATH];
 HANDLE hellfire_mpq;
 char patch_rt_mpq_path[MAX_PATH];
 WNDPROC CurrentProc;
-HANDLE diabdat_mpq;
+HANDLE diabdat_mpq = NULL;

```

Comment:
isn't this initialized as zero/NULL?

Reply:
It's not initialized and if it fails to load the crash dialog is called before this is set to null

Reply:
Although maybe I'm wrong, cause it's static, I'm not sure